src/css/profile/mobile/common/popup.less

```diff
@@ -93,6 +88,144 @@
 	}
 }
 
+@media (min-width: 0px) and (max-width: 479px) {
```

Comment:
I think it would be good to apply the media query only to areas that required separation.

Reply:
good idea. 
however, as the existing popup works well, the spec document should be changed. 
therefore, later when the change is complete, I will separate out the common parts.

Reply:
I modified.

done.